proto/tpm_enrollz.proto

```diff
@@ -58,6 +58,10 @@ message RotateOIakCertRequest {
   // PEM-encoded owner initial DevID certificate signed by the
   // switch owner/administrator CA.
   string oidevid_cert = 3;
+
+  // SSL profile for which the certs will be assigned.
```

Comment:
How about: "Optional SSL profile ID for which the new Owner IDevID certificate should be assigned. This will only be set when a new owner IDevID cert is specified.".

This would clarify that (1) Owner IAK cert is not part of the SSL profile as that cert is only used for attestation (as opposed to TLS) and (2) the field is optional because it won't be set in scenarios where we only rotate Owner IAK cert (but leave Owner IDevID cert as is).